docs/connecting-data/connecting-to-apis-and-external-data/affinity-api.md

```diff
@@ -4,4 +4,148 @@ description: How to Connect DataDistillr to the Affinity API
 
 # Affinity API
 
-Details coming soon!
\ No newline at end of file
+### Cost
```

Comment:
Create a block with this like was done in ClickUp PR

Reply:
I am waiting for the merge because it's a custom admonition